### PR TITLE
Apply temporary background video for title and tutorial dialog

### PR DESCRIPTION
**Pull request by LiteHell**
It uses [Seoul Korea - Free Paper Style Background Loop](https://www.youtube.com/watch?v=6G_v0eLtbRE).

## Screenshots
Title preview

Tutorial dialog preview

### CODE REVIEW

**Review by firekann — APPROVED** [review]
LGTM

**Review by firekann — APPROVED** [review]
LGTM

File: game/src/game/tutorial.rs [hunk 1]
```diff
@@ -2,12 +2,18 @@ mod ending;
 mod greetings;
 mod learn_stick_note;
 
-use std::time::{Duration, Instant};
+use std::{
+    path::Path,
+    time::{Duration, Instant},
```
Comment (boulce) [inline game/src/game/tutorial.rs:7]: time::Duration is not used

**Review by boulce — CHANGES_REQUESTED** [review]

**Review by boulce — APPROVED** [review]
LGTM

**Pull request merged**